src/Pages/OfficerDB/InfoRow.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import 'bootstrap/dist/css/bootstrap.min.css'
+import { Modal, Button, ModalHeader, ModalBody } from 'reactstrap'
+import Editor from './Edits/Editor'
+
+const api = require('../../APIFunctions/OfficerDB')
```

Comment:
lets not mix require with imports. instead, you can directly import the functions [like this](https://github.com/SCE-Development/Core-v4/blob/b4707f9f5ef75f3806b7cc6d2cce5f81454e90f6/src/Pages/Events/EventInfoModal.js#L12)